### PR TITLE
Refs #36090 - Use installer support to remove REX cockpit

### PR DESCRIPTION
**Pull request by ekohl**
There is now proper support to disable the REX cockpit plugin. This is in the form of the ensure parameter. It also modifies the enable command since the installer may have remembered the ensure=absent part.

I'd like QE to verify and ACK this before we merge it.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.10
* [x] Foreman 3.9/Katello 4.11 (planned Satellite 6.15)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* [ ] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [ ] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8; orcharhino 6.3 on EL7/8)
* We do not accept PRs for Foreman older than 3.1.

### CODE REVIEW

**Review by adamlazik1 — APPROVED** [review]
LGTM

**Review by pondrejk — APPROVED** [review]

**Review by asteflova — CHANGES_REQUESTED** [review]

File: guides/common/modules/proc_disabling-cockpit-on-project.adoc
Comment (ekohl) [reply guides/common/modules/proc_disabling-cockpit-on-project.adoc:8]: Good point. Updated, but I wonder: does it even need a bullet?

Comment (asteflova) [reply guides/common/modules/proc_disabling-cockpit-on-project.adoc:8]: The style guide says it does need a bullet so I'd recommend to go with that. Also, it would be consistent with the [module on enabling cockpit](https://github.com/theforeman/foreman-documentation/blob/f11d45fcd533e60dd3bbfb963dcefff2df699d9f/guides/common/modules/proc_enabling-cockpit-on-server.adoc?plain=1#L8).

File: guides/common/modules/proc_disabling-cockpit-on-project.adoc
Comment (asteflova) [inline guides/common/modules/proc_disabling-cockpit-on-project.adoc:7]: Actually, looking at the procedure for enabling cockpit, it has a more informative procedure step description:

`Enable RHEL web console on your Satellite Server:`

vs

`Run this foreman-installer command`

Can you think of a similar way to rephrase this step as well? Looking at the actual command here, I can't tell if simple "Disable RHEL web console on your Satellite Server:" would be accurate.

File: guides/common/modules/proc_disabling-cockpit-on-project.adoc
Comment (asteflova) [inline guides/common/modules/proc_disabling-cockpit-on-project.adoc:13]:
```suggestion

```

With `+`, the IMPORTANT box is part of the step to run foreman-installer. I'm not sure if that's something users would notice but it affects the indentation on the page.

**Review by asteflova — CHANGES_REQUESTED** [review]

File: guides/common/modules/proc_disabling-cockpit-on-project.adoc
Comment (ekohl) [reply guides/common/modules/proc_disabling-cockpit-on-project.adoc]: There's quite a bit of overlap with line 4. I'll just rephrase this line now, but leave it up to other writers to decide what's best here in the long term.

**Review by asteflova — APPROVED** [review]

File: guides/common/modules/proc_disabling-cockpit-on-project.adoc
Comment (asteflova) [reply guides/common/modules/proc_disabling-cockpit-on-project.adoc]: We discussed this at today's meeting and agreed that `Disable {Cockpit} on your {ProjectServer}:`, as included in the latest version of this PR, is okay.

**Comment by asteflova** [comment]
Hi @ekohl, with QE and TW acks in place, is this good to merge? Let me know if you need any further help with wording, structure, or just someone to act as a debugging duck.

**Comment by asteflova** [comment]
Discussed at upstream docs meeting, ready to merge, I'll take care of it.

**Comment by asteflova** [comment]
Merged to "master" and cherry-picked:

d442759282..e5324c7004  3.10 -> 3.10
50a29c7d41..0f802ec699  3.9 -> 3.9

**Pull request merged**
